app/tests/test_admin_page.py

```diff
@@ -220,11 +222,26 @@ def _create_asset_minimal(self, item):
 
         return asset, data
 
-    def _create_asset(self, item, extra=None):
+    def _create_asset(self, item, extra=None, large_file=False):
         start = time.time()
-        filecontent = b'mybinarydata'
-        filelike = BytesIO(filecontent)
-        filelike.name = 'testname.tiff'
+        if large_file:
+            logger.warning(
+                "Running test for admin page with very large asset file. "
+                "This can take a while (about 1.5 minutes), sorry."
+            )
+            asset_file = './tests/sample_data/large_asset_file.tiff'
+            if not os.path.isfile(asset_file):
+                with open(asset_file, "wb") as dummy_file:
+                    dummy_file.seek(1073741824 - 1)
+                    dummy_file.write(b'\0')
```

Comment:
maybe add a comment about how you get to this number

Reply:
Sounds like a good idea, I'll give it a try.
If it works, it would be much better and the resources for the build instance could be reduced to "normal" again :-)

Reply:
Now this whole problem is fixed in a much shorter way